opensearch-dashboards-2.yaml

```diff
@@ -122,6 +134,7 @@ test:
     contents:
       packages:
         - busybox
+        - ${{package.name}}-config
```

Comment:
looks like we dont support templating here :(

Reply:
this should fix that https://github.com/chainguard-dev/melange/pull/1037